EmberFest teaser write-up (#1039)
ember-learn/ember-blog: content/the-ember-times-issue-189.md

---

## [🏛️ EmberFest 2021 - best of both worlds](https://emberfest.eu)

> Next week after a year of absence because of COVID [EmberFest](https://emberfest.eu) is back. On the 30th of September and the 1st of October Emberistas will gather **digitally** and **analogue** in Rome to hear awesome Ember-talks.

Reviewer: Content looks good. I updated a few punctuation marks to help with understanding each sentence.

```suggestion
Next week, after a year of absence because of COVID, [EmberFest](https://emberfest.eu) is back. On the 30th of September and the 1st of October, Emberistas will gather **digitally** and **on-site** in Rome to hear awesome Ember talks.
```

> With well-known EmberConf/Fest speakers like [Melanie Sumner (@MelSumner)](https://github.com/MelSumner) and [Edward Faulkner (@ef4)](https://github.com/ef4) and first time speakers like [Bartlomiej Dudzik (@BobrImperator)](https://github.com/BobrImperator), [Jenny Judova (@JennyJudova)](https://github.com/JennyJudova) and [Zoë Bijl (@ZoeBijl)](https://github.com/ZoeBijl) the [line up](https://emberfest.eu/schedule/) is super diverse and interesting for all Ember enthousiasts.

Reviewer: I think it'll help to break a long sentence into two. How about:

```suggestion
There will be well-known EmberConf/Fest speakers like [Melanie Sumner (@MelSumner)](https://github.com/MelSumner) and [Edward Faulkner (@ef4)](https://github.com/ef4), as well as first-time speakers like [Bartlomiej Dudzik (@BobrImperator)](https://github.com/BobrImperator), [Jenny Judova (@JennyJudova)](https://github.com/JennyJudova), and [Zoë Bijl (@ZoeBijl)](https://github.com/ZoeBijl). The [line-up](https://emberfest.eu/schedule/) is super diverse and interesting for all Ember enthusiasts.
```

Author: Great suggestion :)

> There are still remote tickets available and even if you plan on coming last minute to Rome a few on-site tickets remain. We hope you enjoy this conference wherever you decide to join.

Reviewer: I made a few changes as a suggestion below. What do you think about:

```suggestion
Remote tickets are still available. A few on-site tickets too, if you plan on arriving at the last minute. We hope you enjoy EmberFest wherever you decide to join.
```

?
